frontend/src/components/Blog.jsx

```diff
@@ -0,0 +1,10 @@
+// eslint-disable-next-line react/prop-types
+function Blog({ index, title, content }) {
+  return (
+    <div key={index}>
```

Comment:
key should be declared here. your div has a title : it should be an article or a section